apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -220,7 +220,7 @@ private async Task<int> ExecuteDmlAsync(CancellationToken cancellationToken)
                 ExecuteSqlRequest request = GetExecuteSqlRequest();
                 long count = await transaction.ExecuteDmlAsync(request, cancellationToken, CommandTimeout).ConfigureAwait(false);
                 // This cannot currently exceed int.MaxValue due to Spanner commit limitations anyway.
-                return checked((int)count);
+                return checked((int) count);
```

Comment:
nit: This is just whitespace fix and I see one more below but going forward, we should have a separate commit for any styling fixes.